develop/CICD/Tutorials/CICD_Tutorial_Connector.md

```diff
@@ -0,0 +1,298 @@
+---
+uid: CICD_Tutorial_Connector
+---
+
+# Setting up a GitHub workflow for connectors
+
+In this tutorial, you will learn how to set up basic quality control and automatic deployment of a DataMiner connector to a staging system through a CI/CD pipeline. This can be done with or without the staging system being connected to dataminer.services. This tutorial uses a DataMiner Agent on an internet-accessible virtual machine.
+
+You can find a quick overview of specific CI/CD tooling offered by Skyline Communications in our [documentation](xref:Platform_independent_CICD).
+
+> [!NOTE]
+> You can also apply the instructions in this tutorial (with limited syntax changes) to use other CI/CD technology, such as Jenkins, GitLab, Concourse, Azure DevOps, etc.
+
+Expected duration: 20 minutes.
+
+## Prerequisites
+
+- Runtime .NET SDK 8.0. ([download](https://dotnet.microsoft.com/en-us/download/dotnet/8.0))
+
+- A staging DataMiner Agent that is accessible from your pipeline and that uses DataMiner version 10.3.0/10.3.2 or higher
+
+- [Microsoft Visual Studio](https://visualstudio.microsoft.com/downloads/)
```

Comment:
Is there a minimum version of Visual Studio for this tutorial?

Reply:
After discussing it with @janstaelensskyline , it turns out this procedure is for VS 2022.

Reply:
Could you add this to this prerequisite?